dashboard/src/components/reviewers/ReviewSection.vue

```diff
@@ -119,58 +131,59 @@ const reviewData = createResource({
 })
 
 const handleReviewSubmit = () => {
-    if(!selectedReviewOption.value){
-        errors.value = 'Please select a review option'
-        return
-    }
+  if (!selectedReviewOption.value) {
+    errors.value = 'Please select a review option'
+    return
+  }
 
-    if (reviewData.data){
-        updateReview()
-        return
-    }
+  if (reviewData.data) {
+    updateReview()
+    return
+  }
 
-    submitReview()
+  submitReview()
 }
 
 const updateReview = () => {
-    createResource({
-        url: 'frappe.client.set_value',
-        makeParams() {
-            return {
-                doctype: 'FOSS Event CFP Review',
-                name: reviewData.data.name,
-                fieldname: {
-                    to_approve: selectedReviewOption.value.value,
-                    remarks: newRemarks.value,
-                },
-            }
-        },
-        auto: true,
-        onSuccess() {
-            in_review_edit.value = false
-            reviewData.fetch()
+  createResource({
+    url: 'frappe.client.set_value',
+    makeParams() {
+      return {
+        doctype: 'FOSS Event CFP Review',
+        name: reviewData.data.name,
+        fieldname: {
+          to_approve: selectedReviewOption.value.value,
+          remarks: newRemarks.value,
         },
-        onError(error) {
-            errors.value = 'Failed to update review'
-        },
-    })
+      }
+    },
+    auto: true,
+    onSuccess() {
+      in_review_edit.value = false
+      reviewData.fetch()
+    },
+    onError(error) {
+      errors.value = 'Failed to update review'
+    },
+  })
 }
 
 const submitReview = () => {
-    createResource({
-        url: 'fossunited.api.reviewer.submit_review',
-        makeParams(){
-            return {
-                submission_id: route.params.talk_id,
-                to_approve: selectedReviewOption.value.value,
-                remarks: newRemarks.value,
-            }
-        },
-        auto: true,
-        onSuccess() {
-            in_review_edit.value = false
-            hasReviewed.fetch()
-        },
-    })
+  createResource({
+    url: 'fossunited.api.reviewer.submit_review',
+    makeParams() {
+      return {
+        submission_id: route.params.talk_id,
+        to_approve: selectedReviewOption.value.value,
+        remarks: newRemarks.value,
+        reviewer: session.user,
```

Comment:
looks like the `submit_review` function uses `frappe.session.user` as the default value for `reviewer` - why is that value set to `None` on the dashboard?

should we remove the default value and require `reviewer` value to always be passed?

Reply:
Sorry, `None` where?